vm/vm.go

```diff
@@ -188,36 +191,48 @@ func (vm *VM) GetREPLResult() string {
 	return ""
 }
 
+func (vm *VM) initMainObj() *RObject {
+	return &RObject{Class: vm.builtInClasses[objectClass], InstanceVariables: newEnvironment()}
+}
+
 func (vm *VM) initConstants() {
-	vm.constants = make(map[string]*Pointer)
-	constants := make(map[string]*Pointer)
+	cClass := initClassClass()
+	objClass := initObjectClass(cClass)
 
-	builtInClasses := []Class{
-		integerClass,
-		stringClass,
-		booleanClass,
-		nullClass,
-		arrayClass,
-		hashClass,
-		classClass,
-		methodClass,
-		channelClass,
-	}
+	vm.builtInClasses[classClass] = cClass
+	vm.builtInClasses[objectClass] = objClass
 
-	args := []Object{}
+	constants := make(map[string]*Pointer)
 
-	for _, arg := range vm.args {
-		args = append(args, initStringObject(arg))
+	builtInClasses := []*RClass{
+		vm.initIntegerClass(),
+		vm.initStringClass(),
+		vm.initBoolClass(),
+		vm.initNullClass(),
+		vm.initArrayClass(),
+		vm.initHashClass(),
+		vm.initRangeClass(),
+		vm.initMethodClass(),
+		vm.initializeChannelClass(),
 	}
 
+	vm.initErrorClasses()
+
 	for _, c := range builtInClasses {
 		p := &Pointer{Target: c}
 		constants[c.ReturnName()] = p
```

Comment:
We currently store built in classes in both vm's `constants` and `builtInClasses`, I'll fix this in #224